### ISSUE TITLE
Infrastructure

### ISSUE THREAD

**Issue by mandily-p** [BCcampus/edehr#65]
- [x] Unit tests

- [x] With George at BCcampus

- [x] Move EdEHR to BCcampus

- [x] Fully containerize the services

Moved "Browser stack" to it's own issue.

**Comment by bryan-gilbert** [comment]
- place server into docker image
- create easy one / two line start command

**Comment by mandily-p** [comment]
@bryan-gilbert Do you have any blockers on this ticket?

**Comment by bryan-gilbert** [comment]
Removed the Jenkins (CI) task to post MVP. It's a bigger fish than I anticipated.  Probably be best to look at GitLab.
Added unit tests task and marked it done (for now).  Recently I updated all the unit tests on both the client and server.  These tests aren't close to full coverage but they are now in the best physical location (close to the actual code) and there are many more tests that deal with many of the gnarly bits.
